env.faucet.config.json

```diff
@@ -42,43 +22,34 @@
         "default": "unset",
         "type": "string"
       },
+      "PORT": {
+        "description": "the port you want the server to listen on (should be part of the bot's `SMF_CONFIG_BACKEND_URL`.)",
+        "default": 5555,
+        "type": "number"
+      },
       "EXTERNAL_ACCESS": {
```

Comment:
@mutantcornholio i don't see this config is somewhere used in https://gitlab.parity-mgmt.parity.io/infrastructure/cloud-infra/-/merge_requests/1481/diffs

do we still need this?

Reply:
`EXTERNAL_ACCESS`? Yeah, I'm eager to kill it, but not right now